build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
+  exec program='cmd' commandline='/C dnu commands install --source ${DNX_TOOLS_FEED} NugetPackageVerifier --packages ${BASE_DIR}\commands\' if='ShouldVerifyNupkgs'
```

Comment:
`${BASE_DIR}\commands\' => `Path.Combine`. Also does Sake let you split lines? Kinda hard to read long scrolling lines.